images/git-serve/templates/git.tmpl

```diff
@@ -6,14 +6,22 @@ server {
  
   server_name {{getenv "SERVER_NAME"}};
  
- location = / {
+  location = / {
     index index.html;
   }
 
   location = /index.html {
     index index.html;
   }
 
+  location /repos/ {
```

Comment:
This isn't working for some reason... I would expect any path with a `/repos/...` prefix to be forwarded to the proxy endpoint `<proxy>/repos/...`